src/app/lib/config/services/arabic.js

```diff
@@ -205,6 +205,12 @@ export const service = {
           href: 'https://www.bbc.com/arabic/institutional-37731355',
           text: 'اتصل بـ بي بي سي',
         },
+        {
+          href:
+            'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+          text: 'AdChoices / Do Not Sell My Info',
+          lang: 'en-GB',
```

Comment:
@FK78 I think the lang: 'en-GB' should just apply to English and Scotland.

I also think the text `'AdChoices / Do Not Sell My Info'` might be translated for non English services cc @jimjohnsonrollings